etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/database/PostgresDaoTest.groovy

```diff
@@ -137,4 +137,59 @@ class PostgresDaoTest extends Specification {
         then:
         thrown(SQLException)
     }
+
+    def "fetchMetadata happy path with hasValidData:true works"() {
```

Comment:
Could we update the test names to be more business use case specific? Like "fetchMetadata succeeds when rows exist to return"?

Reply:
Yes, we can.

Reply:
Done.